CMakeLists.txt

```diff
@@ -35,21 +42,29 @@ endif()
 
 include(CMakeDependentOption)
 option(TUVX_ENABLE_MPI "Enable MPI parallel support" OFF)
-cmake_dependent_option(TUVX_ENABLE_OPENMP "Enable OpenMP support" OFF "TUVX_ENABLE_MPI" OFF)
+cmake_dependent_option(TUVX_ENABLE_OPENMP "Enable OpenMP support" OFF
+                       "TUVX_ENABLE_MPI" OFF)
 option(TUVX_ENABLE_TESTS "Build tests" ON)
+option(TUVX_ENABLE_BENCHMARK "Build benchmark examples" ON)
```

Comment:
Do we mean to set this option `ON`as default?

Reply:
I'm not sure. Maybe this is a question for @sjsprecious @mattldawson @K20shores ?

Reply:
I think this should probably be off by default. Users will probably not be interested in running the benchmark tests

Reply:
@mattldawson  This seems to be off by default (even though it set to be on in the cmake file). I had to pass in `- D TUVX_ENABLE_BENCHMARK=ON` for cmake to generate the build files for the benchmarking code.

Reply:
@boulderdaze I turned it off in my latest commit.